pkg/framework/test/integration/integration_suite_test.go

```diff
@@ -23,8 +24,18 @@ var (
 )
 
 var _ = BeforeSuite(func() {
-	assetsDir, ok := os.LookupEnv("KUBE_ASSETS_DIR")
-	Expect(ok).To(BeTrue(), "Expected $KUBE_ASSETS_DIR to be set")
+	assetsDir := ""
+
+	if dirFromEnv, ok := os.LookupEnv("KUBE_ASSETS_DIR"); ok {
```

Comment:
I'm still not a big fan of `KUBE_ASSETS_DIR` rather than individual environment variable for each binary.